colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -113,25 +122,40 @@ object TransferEncoding {
     val value = "chunked"
   }
 
-  case object Gzip extends TransferEncoding {
+  private val all = Seq(Identity, Chunked)
+  def unapply(str : String) : Option[TransferEncoding] = {
+    all.find(_.value == str.toLowerCase)
+  }
+}
+
+sealed trait ContentEncoding {
+  def value: String
+}
+object ContentEncoding {
```

Comment:
ContentEncoding...are we planning on writing pipes/implementations for handling Accept Encoding on http requests?  Would be cool...but for a later date :)